ocs_ci/deployment/helpers/helpers.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
Can we rename this file to `mcg.py` `mcg_helpers.py`, or something similar? I think `helpers.py` is too generic and I would like to avoid another `utils.py` situation from happening.

Reply:
Done